### ISSUE TITLE
Unclear about testing

### ISSUE THREAD

**Issue by Canis-Lupis** [MicrosoftDocs/OfficeDocs-O365ITPro#744]
From the instructions here, it appears that once you set baseline access policies, then MFA is instantly turned on for everyone in the tenant.  We don't want to do that--we want to test with a few people first and then add users in groups after that.  Do I really need to manually add 1,000 users, one at a time, in the "Do you want to exclude any one from these policies" step?  Or is there another way to enable MFA for only a select few?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ed95865-eddb-4ad1-d2e2-5bd618151c1b
* Version Independent ID: ef32bb26-e78a-caad-5b41-d7e49f6ef9b4
* Content: [Set up multi-factor authentication for Office 365 users](https://docs.microsoft.com/en-us/office365/admin/security-and-compliance/set-up-multi-factor-authentication?view=o365-worldwide#enable-multi-factor-authentication-for-your-organization)
* Content Source: [Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md](https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/blob/public/Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md)
* Service: **o365-administration**
* GitHub Login: @Sirkkuw
* Microsoft Alias: **sirkkuw**

**Comment by mypil** [comment]
@Canis-Lupis - Thank you for submitting feedback.

I will get this issue over to the O365ITPro writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by webstaff** [comment]
This is all so unclear and all needs rewriting if you ask me.

2FA and MFA are different things and MS needs to make this clear.
2FA is what you think it is 2 factors of authentication.
MFA is one method of using 2FA. However MFA doesn't have to be 2FA just the new login flow/process.

MFA for all the web apps works fine for the most part.
You put in your username, then password. then you get prompted for your second auth option via web page.

MFA for outlook is a mess, MFA for outlook should look similar to the web MFA / 2FA where instead of a windows style authentication box you get a little web page style login with full 365 branding. If you get that winner.
If you don't and you get old style login dialogue then welcome to the rabbit hole.
Now it could be a bug and might just need to clear the existing windows credentials from the credentials manager or it might be the sign up date on your office 365 tenant was before the date at which MFA was enabled so you need to tell office to tell outlook to use MFA which will setup outlook to use MFA (modern authentication).
"Set-OrganizationConfig -OAuth2ClientProfileEnabled $true" should toggle all outlook 2016 apps to use MFA and older 2013 ones can have a registry option set to allow MFA is its upto date.
Most of the above then applies to other apps to if MFA can be used.
Long story short, if you get a webpage looking 365 branded login option that's MFA, if 2FA is enabled then after your username and password goes in, next screen is second auth option.
If its anything other than that just use an App password.
https://account.activedirectory.windowsazure.com/Proofup.aspx
then click the tap for "app passwords". (which isn't obvious its a clickable toggle!!!!!)

With regards to who and how.
Just enable MFA for all using the powershell option above than means the clients should then try to use modern Auth regardless of is 2FA is implemented.
Then from your azure panel you can toggle 2FA on for individual users, or use the bulk upload option, or script something via powershell.
The easiest way to do individuals is to login to the 365 portal > users > then use the multifactor authentication option from the top of the users list.
At that point once the user add's their 2FA details they should then be up and running with 2FA and modern Auth and should get the office 365 branded login pages in any MFA supported apps which use 2FA.
If they don't it's a bug, get a ticket and expect no answer ;)

**Comment by webstaff** [comment]
Sorry slight correct even I'm doing it wrong now.
I've confused MFA and MA (Modern Authentication)
Modern Authentication with MFA / 2FA which if you jump through the right hoops is what outlook should support and offers the best security.
However currently as its so buggy support suggest using app passwords.

**Comment by get-itips** [comment]
Hello @webstaff 
Thanks for your feedback. 
I recommend looking this article to enable MFA for certain users using Powershell https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-mfa-userstates

To my understanding, the other feedback that you give would be better suited for UserVoice, where that kind of feature requests can be seen by developers and managers of the products please kindly visit https://office365.uservoice.com/

Hope this information helps you

**Comment by webstaff** [comment]
@get-itips
Thanks for the info, already in our internal documentation as fall back info, however it was Canis-Lupis who was asking about the enabling disabling.
For them I suggested make sure toggle on MA via powershell, then from the azure web gui.
Just enable MFA for all using the powershell option above than means the clients should then try to use modern Auth regardless of is 2FA is implemented.
Then from your azure panel you can toggle 2FA on for individual users, or use the bulk upload option, or script something via powershell.
The easiest way to do individuals is to login to the 365 portal > users > then use the multifactor authentication option from the top of the users list.
At that point once the user add's their 2FA details they should then be up and running with 2FA and modern Auth and should get the office 365 branded login pages in any MFA supported apps which use 2FA.

My issue is with the documentation and the confusion within the whole process so many potential paths. Which having spoken to support a couple of times about random issues is as clear as mud still as so much of this is fluid, not sure how requesting more changes via user voice would help that.
They need to pick a process get everyone on the same process / platform and stop and breath for awhile.

But we've condensed this internally down to.
powershell in and make sure modern auth is enabled even if you think it is still run the command to be sure.
make sure the user knows MA vs old auth using pictures to show the two types at least from an outlook POV.
then we have a much clearer layed out support document taking them through the expected path for MFA and when to use app password if MA doesn't do its thing.
Now i've written that this week so I expect it will be wrong by next week. :)

**Comment by webstaff** [comment]
Done it again.
Just enable MA* for all using the powershell not MFA. why did they call it MFA not just 2FA!!!

**Comment by get-itips** [comment]
Hello @webstaff you are right, I was meant to mention @Canis-Lupis in the first part of my other message, sorry. And I hope the other guidelines i made for you are helpful to you.
As the question of the original issue poster is replied, we would proceed closing this one. I encourage you again to open an user voice with your inquiries or open another issue with what you see needs to be corrected **in the documentation** of the MFA feature.
**Thank you**

**Comment by mypil** [comment]
@Canis-Lupis - From our understanding, the issue you raised has been answered by get-itips. If you feel it hasn't been answered, please re-open this issue.

Thank you for your contribution to make the docs better! Much appreciated!

**Comment by Canis-Lupis** [comment]
I’m not sure my original question was answered or perhaps I just missed it in all the back and forth.  I originally stated, “From the instructions here, it appears that once you set baseline access policies, then MFA is instantly turned on for everyone in the tenant.”  Since I posted this question, I have turned on MFA and set a baseline access policy only to get users complaining that they were getting messages to authenticate when I hadn’t done anything to enable it for individual users nor had they even registered.

So, I guess I answered my own question: yes, turning on a baseline policy instantly turns MFA on for everyone.  I think this article should be updated to CLEARLY state this so people don’t create the same problem I did.  Only after turning off baselines policies was I able to assign MFA to only the individuals I wanted to assign it to and have them register.  Baseline policies aren’t necessary for MFA.

**Comment by kenwith** [comment]
@Canis-Lupis - which part of the doc were you in when a note like you suggested would have helped? 

Are you thinking a note along the lines of: "Note: Turning on a baseline policy instantly turns MFA on for everyone. Only after turning off baselines policies will you be able to assign MFA to individuals users and have them register.  Baseline policies aren’t necessary for MFA."

What we can do is put a Pull Request (PR) together and then see if we can get sign off on it to get it merged. Thanks for taking the time to make the docs better too. I know it is easier to just move on once you got it worked out but taking the time to go back and update the docs based on your experience really helps the community and others that will surely run into the same questions you encountered.

**Comment by kenwith** [comment]
@Canis-Lupis - I got in touch with the product team and they are currently changing the behavior to improve the process. This doc will be updated with that new behavior soon. So I will go ahead and close this and we can revisit when the product update happens.
